projects/ngx-mqtt/ng-package.json

```diff
@@ -1,7 +1,8 @@
 {
   "$schema": "../../node_modules/ng-packagr/ng-package.schema.json",
   "dest": "../../dist/ngx-mqtt",
+  "whitelistedNonPeerDependencies": ["mqtt"],
```

Comment:
this one needed due to https://github.com/ng-packagr/ng-packagr/blob/master/docs/dependencies.md#whitelisting-the-dependencies-section